src/components/calcite-date-month-header/calcite-date-month-header.e2e.ts

```diff
@@ -0,0 +1,58 @@
+import { newE2EPage } from "@stencil/core/testing";
+
+describe("calcite-date-month-header", () => {
+  const localeDataFixture = {
```

Comment:
you could just import an existing locale as json, right?

Reply:
I tried that before, but I think the test build trips up with the `fetchAssets` call inside of the date util.